addusers.py

```diff
@@ -130,10 +134,9 @@ def create_project(self, project, quotas):
 
     def create_user(self, user, project):
         """Create a new OpenStack user from an addusers.User
-
```

Comment:
i fixed this. 

you're right i can have the blank line. i don't remember why i deleted it.. :/